demod/mod/rs41mod.c

```diff
@@ -1716,17 +1720,28 @@ static int print_position(gpx_t *gpx, int ec) {
                     if ((!err && !err1 && !err3) || (!err && encrypted)) { // frame-nb/id && gps-time && gps-position  (crc-)ok; 3 CRCs, RS not needed
                         // eigentlich GPS, d.h. UTC = GPS - 18sec (ab 1.1.2017)
                         fprintf(stdout, "{ \"type\": \"%s\"", "RS41");
-                        fprintf(stdout, ", \"frame\": %d, \"id\": \"%s\", \"datetime\": \"%04d-%02d-%02dT%02d:%02d:%06.3fZ\", \"lat\": %.5f, \"lon\": %.5f, \"alt\": %.5f, \"vel_h\": %.5f, \"heading\": %.5f, \"vel_v\": %.5f, \"sats\": %d, \"bt\": %d, \"batt\": %.2f",
-                                       gpx->frnr, gpx->id, gpx->jahr, gpx->monat, gpx->tag, gpx->std, gpx->min, gpx->sek, gpx->lat, gpx->lon, gpx->alt, gpx->vH, gpx->vD, gpx->vV, gpx->numSV, gpx->conf_cd, gpx->batt );
-                        if (gpx->option.ptu && !err0) {
-                            float _RH = gpx->RH;
-                            if (gpx->option.ptu == 2) _RH = gpx->RH2;
+                        fprintf(stdout, ", \"frame\": %d, \"id\": \"%s\", \"datetime\": \"%04d-%02d-%02dT%02d:%02d:%06.3fZ\", \"lat\": %.5f, \"lon\": %.5f, \"alt\": %.5f, \"vel_h\": %.5f, \"heading\": %.5f, \"vel_v\": %.5f, \"sats\": %d, \"bt\": %d, \"batt\": %.2f",gpx->frnr, gpx->id, gpx->jahr, gpx->monat, gpx->tag, gpx->std, gpx->min, gpx->sek, gpx->lat, gpx->lon, gpx->alt, gpx->vH, gpx->vD, gpx->vV, gpx->numSV, gpx->conf_cd, gpx->batt );
+                        fprintf(stdout, ", \"pDOP\": %.1f, \"sAcc\": %.1f",gpx->pDOP,gpx->sAcc);
+
+                    if (gpx->option.ptu && !err0) {
+                            float rh = gpx->RH;
+                            if (gpx->option.ptu == 2  && gpx->RH2 > 0) {
+                                rh = gpx->RH2;
+                            }
```

Comment:
This would output RH, if RH2 < 0 and RH > 0,  so you could get mixed json-output for --ptu2, but the output should be either RH or RH2. If _RH > -0.5 is checked later. (I could change the temporary variable name, though, but that's maybe not a reason for a pull request.) 
Seems like your changes are not based on the latest commit?
Then you want to calculate the dew point again for json? Anyway, I don't know if dew point is really desired by auto_rx, and I didn't want to many derived values.